Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -50,20 +38,27 @@
 numWFSkip=200
 # first two sets are the former D3 WF (now removed as redundant)
 # temporary measure to keep other WF numbers the same
-numWFConflict = [[11000,11200],[11400,11600],[20800,21200],[21600,23200],[25000,26000],[50000,51000]]
+numWFConflict = [[11000,11200],[11400,19800],[20000,20400],[20800,23200],[23600,24000],[24400,26200],[26600,27000],[50000,51000]]
 numWFAll={
-    2017: [numWFStart[2017]],
-    2023: [numWFStart[2023]]
+    2017: [],
+    2023: []
 }
 
+
 for year in upgradeKeys:
-    for i in range(1,len(upgradeKeys[year])):
```

Comment:
I think this could be fixed a bit more simply/efficiently:
```
for i in range(0,len(upgradeKeys[year])):
    numWFtmp = numWFStart[year] if i==0 else (numWFAll[year][i-1] + numWFSkip)
```